update: IP allowlist for tokens (#614)
aiven/aiven-docs: docs/platform/howto/set-authentication-policies.md

import ConsoleLabel from "@site/src/components/ConsoleIcons"

> The authentication policy for your organization specifies the ways that users in your organization can access the organization on the Aiven Platform.

Reviewer: ```suggestion
The authentication policy for your organization specifies how users can access the organization on the Aiven Platform.
```
"Organization" is repeated multiple times. Reducing redundancy can improve readability.

Author: I completely understand the suggestion, but the auth policy specifically applies to org users. The complexity around user management now with org users, managed users, along with the fact that users can be part of multiple orgs, means that we need to be specific even if it sounds redundant.
